### ISSUE TITLE
[BUG] Repeater doesn't work at 3.2.1 release

### ISSUE THREAD

**Issue by mstys** [bolt/bolt#6013]
Hi, I'm started today with new version of Bolt (**3.2.1**) and I have following issue.

When I added repeater to my template
```
anotherextrafields.twig:
        features:
            type: repeater
            fields:
                repeattitle:
                    type: text
                repeatimage:
                    type: image
                    extensions: [ gif, jpg, png ]
                repeatcontent:
                    type: html
````

and tried create new page using this template I get error:

```
## Uncaught Exception: ContextErrorException

_ContextErrorException in RepeaterType.php line 85:_ 
_**Notice: Undefined index: repeater**_

```

```
public function hydrate($data, $entity)
    {
        $key = $this->mapping['fieldname'];
        if ($this->isJson($data[$key])) {
            $originalMapping[$key]['fields'] = $this->mapping['fields'];
            $originalMapping[$key]['type'] = 'repeater';
            $mapping = $this->em->getMapper()->getRepeaterMapping($originalMapping);

            $decoded = json_decode($data[$key], true);
```

Using another contenttype's I did't find this issue.

**Comment by bobdenotter** [comment]
Hi, 

Which install method for Bolt are you using? Not the packaged `.zip` or `.tgz` I assume? 

To avoid this, in your `config.yml` set the following: `debug_error_level: 8181`

**Comment by mstys** [comment]
Hi, I used install by composer.

**Comment by bobdenotter** [comment]
Ok. Have you tried my workaround? 

@rossriley Can you take a look at this to fix properly?

**Comment by mstys** [comment]
Yep, adding `debug_error_level: 8181` to config.yml helped.

**Comment by GawainLynch** [comment]
@rossriley was just looking into something I think is related for @nestordedios (c.f. #5403 & #5684) where custom field types used in the `templatefields` are not being correctly registered in the mapping before the `TemplateFileld` objects are built.

**Comment by Maelstromeous** [comment]
Just looked into this myself, with the repeaterset I've got, I've not encountered this issue, I was able to create the page successfully. Bolt 3.2.1.

**Comment by Maelstromeous** [comment]
For reference, my repeater:

```
templatefields:
    who-we-are/index.twig:
        clients:
            type: repeater
            label: Clients
            fields:
                name:
                    type: text
                image:
                    type: image
                    extensions: [gif, jpg, jpeg, png]
```

**Comment by bobdenotter** [comment]
@Maelstromeous 

> Just looked into this myself, with the repeaterset I've got, I've not encountered this issue, I was able to create the page successfully.

The trick is in the level of error reporting.. If you have `debug_error_level: 8181`, it won't show.

**Comment by GawainLynch** [comment]
Fixed in #6031
